mt940_writer.py

```diff
@@ -91,6 +102,8 @@ def __iter__(self):
         yield ':60F:%s' % self.opening_balance
         for transaction in self.transactions:
             yield ':61:%s' % transaction
+            if transaction.additional_info != '':
```

Comment:
might be more idiomatic simply as `if transaction.additional_info:` so that `None` is also allowed as an indicator for “no additional information”

Reply:
Hey @ushkarev , 
I made the changes so that None and empty string would count as no additional info.
Let me know what you think.